Testcontainers.sln.DotSettings

```diff
@@ -0,0 +1,10 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=CheckNamespace/@EntryIndexedValue">HINT</s:String>
```

Comment:
Can we move the Rider / ReSharper related settings from the `.editorconfig` to this file  or the other way around? It would be nice to have all settings in one configuration file:
https://github.com/testcontainers/testcontainers-dotnet/blob/49c876d02260721cc654ee1c5dce9963ce18924d/.editorconfig#L462-L467

Reply:
I will move this